app/models/setting/remote_execution.rb

```diff
@@ -68,7 +68,7 @@ def self.load_defaults
                  N_('When enabled, working directories will be removed after task completion. You may override this per host by setting a parameter called remote_execution_cleanup_working_dirs.'),
                  true,
                  N_('Cleanup working directories')),
-        self.set('remote_execution_cockpit_href',
+        self.set('remote_execution_cockpit_url',
```

Comment:
I've changed `href` to `url`, as it seemed more appropriate to me + matches with other settings, such as `provision_url` or `foreman_url`

Reply:
Yes, I agree.  Initially, I thought that a relative URL like `/webcon/foo` is not in fact a URL, but it is.